src/aiortc/rtcrtpsender.py

```diff
@@ -326,9 +326,8 @@ def _send_keyframe(self) -> None:
         except (asyncio.CancelledError, ConnectionError, MediaStreamError):
             pass
 
-        # stop track
+        # unreference the track
         if self.__track:
-            self.__track.stop()
             self.__track = None
```

Comment:
Actually no need to check it. Just do `self.__track = None` without any previous check.

Reply:
This is unrelated to the problem at hand, let's keep this out of this commit.